src/opnsense/scripts/suricata/lib/downloader.py

```diff
@@ -204,16 +205,13 @@ def download(self, url, url_filename, filename, auth=None, headers=None, version
                         target_filename, fetch_result['filename']
                 ))
                 return None
-            if not fetch_result['cached']:
+            if 'cached' not in fetch_result:
```

Comment:
`cached` is a boolean, in which case the above statement would return true when false.

Reply:
Sorry for the delay on this and thanks you for the review. I've updated the code based on your comment.

Reply:
@btv no problem, we're all busy :)